src/config/contributors.js

```diff
@@ -0,0 +1,15 @@
+export default [
+  'Casey S. Greene, Dongbo Hu',
+  ' Richard W. W. Jones',
+  'Stephanie Liu',
+  'David S. Mejia',
+  'Rob Patro',
+  'Stephen R. Piccolo',
+  'Ariel Rodriguez Romero',
+  'Hirak Sarkar',
+  'Candace L. Savonen',
+  'Jaclyn N. Taroni',
+  'William E. Vauclain',
+  ' Deepashree Venkatesh Prasad',
```

Comment:
Would you please add Ark's and your name in alphabetical order to this list. You can ask him for his preferred spelling / presentation.

Reply:
Updated, thank you!